tests/h/search/query_test.py

```diff
@@ -3,188 +3,182 @@
 
 import pytest
 
-from h import search
+from h.search import Search, query
 
 
 class TestTopLevelAnnotationsFilter(object):
 
-    def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, search):
-        annotation = Annotation(shared=True)
-        reply = Annotation(references=[annotation.id], shared=True)
+    def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, filtered_search):
+        annotation = Annotation()
+        reply = Annotation(references=[annotation.id])
 
-        result = search.run({})
+        result = filtered_search.run({})
 
         assert annotation.id in result.annotation_ids
         assert reply.id not in result.annotation_ids
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        search_ = search.Search(pyramid_request)
-        search_.append_filter(search.TopLevelAnnotationsFilter())
-        return search_
+    def filtered_search(self, search):
+        search.append_filter(query.TopLevelAnnotationsFilter())
+        return search
 
 
 class TestAuthorityFilter(object):
-    def test_it_filters_out_non_matching_authorities(self, Annotation, search):
-        annotations_auth1 = [Annotation(userid="acct:foo@auth1", shared=True).id,
-                             Annotation(userid="acct:bar@auth1", shared=True).id]
+    def test_it_filters_out_non_matching_authorities(self, Annotation, filtered_search):
+        annotations_auth1 = [Annotation(userid="acct:foo@auth1").id,
+                             Annotation(userid="acct:bar@auth1").id]
         # Make some other annotations that are of different authority.
-        Annotation(userid="acct:bat@auth2", shared=True)
```

Comment:
Since we aren't testing with the shared filter anymore we don't need to mark all these as shared annotations.

Reply:
👍